docs/api.rst

```diff
@@ -1300,6 +1300,53 @@ Voice
     :param after: The voice state after the changes.
     :type after: :class:`VoiceState`
 
+AutoMod
+~~~~~~~
```

Comment:
The automod category should be on the top rather than the bottom since the events are sorted in alphabetical order.